src/css/advertisement.css

```diff
@@ -0,0 +1,77 @@
+
+.visually-hidden {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    border: 0;
+    padding: 0;
+    white-space: nowrap;
+    clip-path: inset(100%);
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+}
+.container-discount{
+    padding: 80px 10px 80px 10px;
+    width: 375px;
+    height: 469px;
+}
+
+.text-discount{
+    font-weight: 600;
+    font-size: 48px;
+    line-height: 1.04;
+    letter-spacing: -0.01em;
+    text-transform: uppercase;
+    color: #fbfbfb;
+    padding: 3px 10px 0 10px;
+    background-color: #576f35;
+    border-radius: 12px;
+    width: 355px;
+    height: 102px;
+}
```

Comment:
line 20-32
fix line height - round up to 1
remove lines 28-31
set up padding: 3px 10px 0;
add z-index: 999;
change color: inherit;